build.proj

```diff
@@ -25,11 +25,11 @@
   </Target>
 
   <PropertyGroup>
-    <OptDataProjectJson>$(SourceDir).nuget/optdata/project.json</OptDataProjectJson>
+    <OptDataCsproj>$(SourceDir).nuget/optdata/optdata.csproj</OptDataCsproj>
     <OptDataPackageFeed>https://dotnet.myget.org/F/dotnet-core-optimization-data/api/v3/index.json</OptDataPackageFeed>
   </PropertyGroup>
   <Target Name="RestoreOptData">
-    <Exec Condition="Exists('$(OptDataProjectJson)')" Command="$(DnuRestoreCommand) &quot;$(OptDataProjectJson)&quot; --source &quot;$(OptDataPackageFeed)&quot;" />
+    <Exec Condition="Exists('$(OptDataCsproj)')" Command="$(DotnetToolCommand) $(DotnetToolArgs) &quot;$(OptDataCsproj)&quot; --source &quot;$(OptDataPackageFeed)&quot;" />
```

Comment:
The old DnuRestoreCommand include other feeds as well, I think that is still needed. Switching this to DotnetRestoreCommand should fix the problem.